### PR TITLE
Project3

### PR DESCRIPTION
**Pull request by sergiogval**
# Last deliver

This PR brings cool features, focused mainly in UX and accessibility. The Icons are functional, also has a feel and look that matches the template very well. I wish I had more time to add some more features.

### CODE REVIEW

File: src/modules/addTodo.js [hunk 1]
```diff
@@ -1,47 +1,44 @@
...
   input.value = '';
+  trash.attachEvent('reset', (e) => {
+    e.preventDefault();
+    todoEl.innerHTML = '';
+    localStorage.clear();
+  });
+  const todos = {};
+  localStorage.setItem(todos, JSON.stringify(todo));
 }
-
```
Comment (Mihndim2020) [inline src/modules/addTodo.js:31]: - [ ] As it is requirement for this project, Implement a function for the `Clear all completed` button (use filter() method).

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @sergiogval, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @sergiogval, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/addTodo.js [hunk 2]
```diff
@@ -1,47 +1,44 @@
...
-  }
-}
-
-Array.from(checkbox).forEach((checkbox) => {
-  checkbox.addEventListener('change', completeTodo);
-  checkbox.addEventListener('change', removeTodo);
-});
+// localStorage
+// .setItem(todos, (JSON.stringify([`${todoEl.id}`,
+// `${todo.description}`, `${checkbox.value}`])));
```
[the remark this answers is not in the document]
Comment (sergiogval) [reply src/modules/addTodo.js:44]: So, I had sent this code for a review just in time to be approved before the deadline and it was rejected with no comments. Now you give me this feed back with a couple of phrases just and 1 minute after deadline? It seems like a big joke to me.

[the remark this answers is not in the document]
Comment (Mihndim2020) [reply src/modules/addTodo.js:44]: Hello @sergiogval, sorry about that, I submitted the review summary right ahead of time before submitting the review. Connection issues may be. I sincerely apologize for any inconveniences caused.

[the remark this answers is not in the document]
Comment (sergiogval) [reply src/modules/addTodo.js:44]: @Mihndim2020 This fail will hit me hard. Now I have to take a leave and start the module from the scratch. Do you think my tech skills aren't good enough to keep going ahead block 3? I recon my responsibility but it just seems unfair from my point of view. Anyway I do appreciate your feedback. Thanks.

**Pull request merged**